db/migrate/20201210090008_create_articles.rb

```diff
@@ -0,0 +1,11 @@
+class CreateArticles < ActiveRecord::Migration[6.0]
+  def change
+    create_table :articles do |t|
+      t.string :title
+      t.text :lead
+      t.text :body
```

Comment:
looking at your PT story you shouldn't have body

Reply:
Okey, this is now fixed!